src/envo/envo-edit.owl

```diff
@@ -40152,7 +40152,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501101> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501102> (polytunnel)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501102> "A tunnel made from steel and covered in polyethylene used for plant cultivation."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501102> "A tunnel made from steel and covered in polyethylene used for plant cultivation."@en)
```

Comment:
@ddooley The definition says "A tunnel ...". But, when I look at the ontology, it is a subclass of building.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/3186638/161871774-532d0b13-0145-4db1-ab2f-cbd6ce979c66.png">

Reply:
I've changed "tunnel" to industrial building, thx.